docs/cristin.ttl

```diff
@@ -434,10 +436,11 @@
 
 :arstall_rapportert
     a                  :cristinfield ;
-    rdfs:comment       "Year that the resource was included in an NVI report." ;
+    rdfs:comment       """Year that the resource was included in an NVI report. For Results of type Book it is used to generate a confirmedPublisher.
```

Comment:
Do not add anything to do with conversion to NVA. Better to use the cucumber scenarios for this. The reason I want to avoid describing the NVA conversion here, is that it is difficult to keep it in sync since this is not testable.